docs/core/responses.rst

```diff
@@ -156,3 +156,31 @@ The endpoint then needs to respond with the generated response:
   }
 
 Rasa will then use this response and sent it back to the user.
+
+
+.. _external-events:
+
+Proactively Reaching Out to the User with External Events
+---------------------------------------------------------
+
+You may want to proactively reach out to the user,
+for example to display the output of a long running background operation
+or notify the user of an external event.
+
+To do so, you can ``POST`` an intent to the
+`trigger_intent endpoint <../../api/http-api/#operation/triggerConversationIntent>`_.
+The intent, let's call it ``EXT_sensor``, will be treated as if the user had sent a message with this intent.
+You can even provide entities as parameters, e.g. ``{"temperature": "high"}``.
+For your bot to respond, we recommend you use the :ref:`mapping-policy` to connect the sent intent ``EXT_sensor``
+with the action you want your bot to execute, e.g. ``utter_warn_temperature``.
+You can also use a custom action here, of course.
+
+Use the ``output_channel`` query parameter to specify which output
+channel should be used to communicate the assistant's responses back to the user.
+Any messages that are dispatched in the custom action will be forwarded to the specified output channel.
+
+.. note::
+
+   Proactively reaching out to the user is dependent on the abilities of a channel and
+   hence not supported by every channel. If your channel does not support it, consider
+   using the :ref:`callbackInput` channel to send messages to a webhook.
```

Comment:
How does that work?

Reply:
I just copied this from the action trigger description, as all input and output is the same, it should work the same and have the same limitations. But I haven't done this webhook thing myself. Who wrote this?

Reply:
Works the same as before. The callBackInput channel simply sends a `POST` request to a predefined HTTP endpoint including the messages as request payload.

So:

User -- message --> Rasa Input Channel
Rasa -- responses --> CallbackOutputChannel -- responses --> HTTP server